master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -2,76 +2,63 @@
+[kubeadm](http://kubernetes.io/docs/getting-started-guides/kubeadm/).  {{site.prodname}}
+is compatible with kubeadm creaetd clusters, as long as the [requirements](#requirements) are met. 
 
-## Installation
+## Requirements
 
-You can easily create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+For {{site.prodname}} to be compatible with your kubeadm-created cluster:
 
-Calico installations require kubeadm initialized clusters to have set a `--pod-network-cidr`
-which should match the value set by `CALICO_IPV4POOL_CIDR` in the manifest:
+* It must be running at least Kubernetes v1.7
 
-```yaml
-- name: CALICO_IPV4POOL_CIDR
-  value: "192.168.0.0/16"
-```
+* There should be no other CNI network configurations installed in /etc/cni/net.d (or equivalent directory)
 
-This value can be changed to any CIDR that does not overlap the cluster's service CIDR.
-The manifests are currently setup to use the CIDR `192.168.0.0/16` so you would initialize
-the cluster with the following command:
+* The kubeadm flag `--pod-network-cidr` must be set when creating the cluster with `kubeadm init` 
+  and the CIDR(s) specified with the flag must match {{site.prodname}}'s IP pools. The default 
+  IP pool configured in {{site.prodname}}'s manifests is `192.168.0.0/16`
 
-```shell
-kubeadm init --pod-network-cidr=192.168.0.0/16
-```
+* The CIDR specified with the kubeadm flag `--service-cidr` must not overlap with 
+  {{site.prodname}}'s IP pools
+  
+  * The default CIDR for `--service-cidr` is `10.96.0.0/12`
+  
+  * The default IP pool configured in {{site.prodname}}'s manifests is `192.168.0.0/16`
 
-### etcd datastore
+You can create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
-Users who have deployed their own etcd cluster outside of kubeadm should
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
+## Installing {{site.prodname}} with a Kubernetes-hosted etcd
 
-To install this Calico and a single node etcd on a run the following command:
+As a non-production quick start, to install {{site.prodname}} with a single-node dedicated etcd cluster,
+running as a Kubernetes pod:
 
-> **Note**: The following manifest requires Kubernetes 1.7.0 or later.
-{: .alert .alert-info}
+1. Ensure your cluster meets the [requirements](#requirements) (or re-create it if not).
 
-```shell
-kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml
-```
+2. Apply the single-node etcd manifest:
+   
+   ```shell
+   kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml
+   ```
+   
+   >[Click here to view the above yaml directly.](1.7/calico.yaml)
```

Comment:
This is not meant to be overly picky but in an ideal world, it would be better if you could use a slightly different format for this....we are trying to standardize on the following instead:

```
   > **Note**: You can also 
   > [view the YAML in your browser](1.7/calico.yaml){:target="_blank"}.
   {: .alert .alert-info}
```